br/pkg/task/show/cmd.go

```diff
@@ -0,0 +1,162 @@
+// Copyright 2023 PingCAP, Inc. Licensed under Apache-2.0.
+
+package show
+
+import (
+	"context"
+
+	"github.com/pingcap/errors"
+	backuppb "github.com/pingcap/kvproto/pkg/brpb"
+	berrors "github.com/pingcap/tidb/br/pkg/errors"
+	"github.com/pingcap/tidb/br/pkg/logutil"
+	"github.com/pingcap/tidb/br/pkg/metautil"
+	"github.com/pingcap/tidb/br/pkg/storage"
+	"github.com/pingcap/tidb/br/pkg/task"
+)
+
+type Config struct {
+	Storage    string
+	BackendCfg storage.BackendOptions
+	Cipher     backuppb.CipherInfo
+}
+
+// lameTaskConfig creates a `task.Config` via the `ShowConfig`.
+// Because the call `ReadBackupMeta` requires a `task.Cfg` (which is a huge context!),
+// for reusing it, we need to make a lame config with fields the call needs.
+func (s *Config) lameTaskConfig() *task.Config {
+	return &task.Config{
+		Storage:        s.Storage,
+		BackendOptions: s.BackendCfg,
+		CipherInfo:     s.Cipher,
+	}
+}
+
+// TimeStamp is a simple wrapper for the timestamp type.
+// Perhaps we can enhance its display.
+type TimeStamp uint64
+
+type RawRange struct {
+	ColumnFamily string           `json:"column-family"`
+	StartKey     logutil.HexBytes `json:"start-key"`
+	EndKey       logutil.HexBytes `json:"end-key"`
+}
+
+type Table struct {
+	DBName         string `json:"db-name"`
+	TableName      string
+	KVCount        uint
+	KVSize         uint
+	TiFlashReplica uint
```

Comment:
miss `json: ...`(otherwise it would use default json name) or `json:"-"`

Reply:
🤣